src/app/community-page/community-page-routes.ts

```diff
@@ -72,9 +72,13 @@ export const ROUTES: Route[] = [
         component: ThemedCommunityPageComponent,
         children: [
           {
-            path: '',
+            path: 'search',
```

Comment:
See comment above.  This also looks odd because it appears to be removing the default path for Communities (e.g. `[dspace.ui.url]/communities/[uuid]`).  Similar to above, testers should verify this doesn't accidentally break links to Community homepages or similar